include/orbit_debug.h

```diff
@@ -0,0 +1,31 @@
+#ifndef ORBIT_DEBUG_H
+#define ORBIT_DEBUG_H
+
+#define ORBIT_DEBUG_RETRIES 3
+#define ORBIT_DEBUG_SUCCESS 0
+#define ORBIT_DEBUG_FAIL 69
+
+
+#include "sci.h"
+
+#include "FreeRTOS.h"
+#include "os_semphr.h"
+#include "os_task.h"
+
+#include "stdlib.h"
+#include "string.h"
+
+typedef struct DebugMsg {
+    char* msg;
+    bool dynamic;
```

Comment:
This should probably be `static` instead.
In most cases DebugMsg will be dynamically allocated and the boolean will deafult to false.
Static declarations should be required to explicitly set this value.